### PR TITLE
added email parser

### PR DESCRIPTION
**Pull request by yuvalherziger**
See https://github.com/mpscholten/request-parser/issues/20

### CODE REVIEW

**Comment by yuvalherziger** [comment]
I'll do the same with emails

**Comment by yuvalherziger** [comment]
ready for review 👍

File: examples/psr7.php [hunk 1]
```diff
@@ -50,6 +51,12 @@ public function json()
 
         return print_r($payload, true);
     }
+
+    public function email()
```
Comment (mpscholten) [inline examples/psr7.php]: Same as in the other PR :)

Comment (yuvalherziger) [inline examples/psr7.php]: done

**Comment by mpscholten** [comment]
@yuvalherziger could you quickly fix the merge conflict? :)

**Comment by yuvalherziger** [comment]
Fixed, pending ci check.

**Pull request merged**